src/app/splashscreen/splashscreen.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="text-center">
+	<H1><span class="titleleft">PC Prep</span><span class="titleright">Kit</span></H1>
```

Comment:
That indentation was default in atom, read HTML style guide, changing it to 2 spaces.